Dockerfile.rhel

```diff
@@ -1,6 +1,5 @@
-FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.12 AS builder
+FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.13 AS builder
 WORKDIR /go/src/github.com/openshift/openshift-controller-manager
-RUN yum install -y gpgme-devel libassuan-devel
```

Comment:
folks looking at this fresh may be unaware, but these RPMs are already in the golang image we use as our builder/base
so this yum install was redundant at a minimum, and an opportunity for mirror flakes to mess up builds.